core/templates/dev/head/filters.js

```diff
@@ -407,6 +409,21 @@ oppia.filter('stripFormatting', [function() {
   };
 }]);
 
+oppia.filter('getAbbreviatedText', [function () {
+  return function (text, characterCount) {
+    if (text.length > characterCount) {
+      var subject = text.substr(0, characterCount);
+        
+      if (subject.includes(' ')) {
```

Comment:
.incudes() isn't supported in FF, I think? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes#compatNote_1

Reply:
Done